rules/highlight.js

```diff
@@ -0,0 +1,105 @@
+const { keyword, maybe_bang, commaSep1 } = require("./utils");
+
+module.exports = {
+  // :h :highlight
+
+  hl_group: ($) => /[a-zA-Z0-9_]+/,
```

Comment:
In neovim `0.8` the `.` and `@` are now valid in highlights groups.

Reply:
...specifically for tree-sitter highlighting ;)

Reply:
How does this interacts with syntax clusters ? Does this means `goup1` here contains the hi group `@g1` or cluster `g1` ?
```vim
syntax cluster group1 contains=@g1
```
Because currently this means group1 contains the cluster `g1`

Reply:
That's a good question.

Reply:
The tests are somehow failing now ?

Reply:
Yes because as the `@` is allowed in the hl_group, tests with syntax clusters are falling because the node `(at)` is not picked up anymore.
It's not a big issue I was just wondering how to handle them: I just remove the nodes in the test ?

Reply:
Just remove this node both in the grammar and in the tests.